app/assets/javascripts/admin/addon/mixins/setting-component.js

```diff
@@ -330,17 +235,17 @@ export default Mixin.create({
   }),
 
   setValidationMessage: action(function (message) {
-    this.set("validationMessage", message);
+    this.setting.set("validationMessage", message);
   }),
 
   cancel: action(function () {
     this.buffered.discardChanges();
-    this.set("validationMessage", null);
+    this.setting.set("validationMessage", null);
   }),
 
   resetDefault: action(function () {
-    this.set("buffered.value", this.setting.default);
-    this.set("validationMessage", null);
+    this.setting.set("buffered.value", this.setting.default);
```

Comment:
We refer to buffered via direct property access elsewhere in the PR so we can be consistent here (and buffered isn't being observed or using `computed` so direct property access should be ok).

```suggestion
    this.setting.buffered.set("value", this.setting.default)
```